src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -72,49 +66,54 @@ A detailed list of the WebExtension APIs supported in Firefox for Android is pro
 
+### Effect on your add-on UI
+
+You cannot expose your add-on through a sidebar or context menu. However, you can expose your extension as a toolbar or address bar button.
+
+Depending on the manifest version your app is using you add a toolbar button using:
+
+- for Manifest V2, the `manifest.json` [`browser_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action) key and [`browserAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/browserAction) API.
+- for Manifest V3, the `manifest.json` [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action) key and [`action`](https://developer.mozilla.org//Add-ons/WebExtensions/API/action) API.
+
+You can also use an address bar button (through the `manifest.json` [`page_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action) key and [`pageAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/pageAction) API) remembering that by default this button is hidden and must be shown programmatically.
+
+The features of [`pageAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction) in Firefox for Android differ slightly from the desktop version. The `manifest.json` key [page_action](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action) lets you tdefine the button icon and a popup. You then have use of [`pageAction.show()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/show) and [`pageAction.hide()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/hide) however, once ‘shown’, the address bar button is visible in all tabs (unlike the desktop behavior, where the button is shown only for a specified tab.) You can hide the pageAction using [`pageAction.hide()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/hide) on a tab (say, for example, you wish to hide your extension's page action icon in `about:addons` or `about:memory` tabs) And you can set a listener to [`pageAction.onClicked()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/onClicked). [`pageAction.setPopup()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/setPopup) and [`pageAction.getPopup()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/getPopup) are also available, so you can update the popup or create a popup once the add-on is running.
+
+Also, in [`pageAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction), [`browserAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/browserAction), and [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/action), popup content is opened in a new tab and persists until the user closes the tab.
```

Comment:
**issue:** the mention related to the popup content opening in a new tab was related to Fennec (the old XUL based mobile browser) while in Fenix the popup content is opening in a "modal overlay" UI, which covers the entire browser, a back arrow in the panel header allow the user to close the popup content UI and get back to the browser UI.

Follows screenshot (in the collapsible details right below).

<details>

![image](https://github.com/mozilla/extension-workshop/assets/11484/96a3993e-897c-4133-bba3-61495f634739)


</details>

Reply:
Thanks, I updated the wording